community/brownout/version

```diff
@@ -1 +1,9 @@
+<<<<<<< HEAD
+<<<<<<< HEAD
 0.4 1
+=======
+git 1
+>>>>>>> b8475701 (brownout: new package at 0.4)
+=======
+0.4 1
+>>>>>>> 5ca5f8cf (Switched sources to tar.gz)
```

Comment:
Your rebasing appears to have been catastrophic. 

[This](https://www.youtube.com/watch?v=kMvLn8WcAII) was on my list of things to watch a while back and you might find it helpful.

Reply:
Yep, I'm getting it now! I'll close this one and resubmit. :)